### PR TITLE
removed broken link added new link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #9295   so I removed broken link and added new link

### CODE REVIEW

File: windows/client-management/mdm/azure-active-directory-integration-with-mdm.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ In both scenarios, the enrollment flow provides an opportunity for the MDM servi
...
 
-For Azure AD enrollment to work for an Active Directory Federated Services (AD FS) backed Azure AD account, you must enable password authentication for the intranet on the ADFS service as described in solution \#2 in [this article](https://go.microsoft.com/fwlink/?LinkId=690246).
+For Azure AD enrollment to work for an Active Directory Federated Services (AD FS) backed Azure AD account, you must enable password authentication for the intranet on the ADFS service as described in solution \#2 in [this article](https://docs.microsoft.com/windows-server/identity/ad-fs/operations/configure-ad-fs-and-azure-mfa).
 
-Once a user has an Azure AD account added to Windows 10 and enrolled in MDM, the enrollment can be manages through **Settings** > **Accounts** > **Work access**. Device management of either Azure AD Join for corporate scenarios or BYOD scenarios are similar.
+Once a user has an Azure AD account added to Windows 10 and enrolled in MDM, the enrollment can be manages through **Settings** > **Accounts** > **Work access**. Device management of either Azure AD Join for corporate scenarios or BYOD scenarios are similar.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/azure-active-directory-integration-with-mdm.md:61]:
```suggestion
Once a user has an Azure AD account added to Windows 10 and enrolled in MDM, the enrollment can be managed through **Settings** > **Accounts** > **Work access**. Device management of either Azure AD Join for corporate scenarios or BYOD scenarios is similar.
```

File: windows/client-management/mdm/azure-active-directory-integration-with-mdm.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ In both scenarios, the enrollment flow provides an opportunity for the MDM servi
 
 In the out-of-the-box scenario, the web view is 100% full screen, which gives the MDM vendor the ability to paint an edge-to-edge experience. With great power comes great responsibility! It is important that MDM vendors who chose to integrate with Azure AD to respect the Windows 10 design guidelines to the letter. This includes using a responsive web design and respecting the Windows accessibility guidelines, which includes the forward and back buttons that are properly wired to the navigation logic. Additional details are provided later in this topic.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/azure-active-directory-integration-with-mdm.md:57]:
```suggestion
In the out-of-the-box scenario, the web view is 100% full screen, which gives the MDM vendor the ability to paint an edge-to-edge experience. With great power comes great responsibility! It is important that MDM vendors who chose to integrate with Azure AD respect the Windows 10 design guidelines to the letter. This includes using a responsive web design and respecting the Windows accessibility guidelines, which includes the forward and back buttons that are properly wired to the navigation logic. Additional details are provided later in this topic.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - a few additional edits, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Good link replacement, thank you for finding the correct URL and making this PR.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**